### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by k9845**

### CODE REVIEW

**Comment by annshress** [comment]
@k9845  gitignore the `.env`. Push `env.sample` or something.

**Comment by annshress** [comment]
Missing `.gitignore`.

**Review by annshress — COMMENTED** [review]
Missing `.gitignore`

**Comment by annshress** [comment]
Apart from the `.env` file, everything looks good.

**Review by annshress — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM python:3.8.2-buster
+
+MAINTAINER togglecorp info@togglecorp.com
```
Comment (thenav56) [inline Dockerfile]: `MAINTAINER` has been deprecated.

We need something like this.
```Dockerfile
LABEL maintainer="Togglecorp Dev dev@togglecorp.com"
```

**Review by thenav56 — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
+__pycache__/
+.env
+
+media/
+
+static/
```
Comment (tnagorra) [inline .gitignore:7]: It's also better to add `*.swp`

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
+FROM python:3.8.2-buster
+
+LABEL maintainer="Togglecorp Dev dev@togglecorp.com"
```
Comment (tnagorra) [inline Dockerfile:3]: Isn't `maintainer` deprecated?

File: chrono/settings.py [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/3.0/howto/static-files/
+
+STATIC_URL = '/static/'
+
+# https://docs.graphene-python.org/projects/django/en/latest/settings/
+GRAPHENE = {
+    'SCHEMA': 'helix.schema.schema',
```
Comment (tnagorra) [inline chrono/settings.py]:
```suggestion
    'SCHEMA': 'chrono.schema.schema',
```

**Pull request merged**